FluentUI.Demo/src/main/java/com/microsoft/fluentuidemo/util/Content.kt

```diff
@@ -98,7 +100,7 @@ fun getAndroidViewAsContent(
                 .fillMaxWidth()
                 .verticalScroll(rememberScrollState()),
             factory = {
-                val view = it.activity!!.layoutInflater.inflate(
+                val view = LayoutInflater.from(it as Activity).inflate(
```

Comment:
Could you explain this change

Reply:
If you check line 105, we are passing 'null' as parent view root during inflation because we do not a specified one. But passing null as parent view root may lead to crashes or unexpected behavior. LayoutInflator.from(it as Activity) uses application context as parent view root and prevents the app from crashing. 
You will find the same change in V2DrawerActicity and V2BotttomSheetActivity because of the same reason.